bundler/lib/dependabot/bundler/update_checker/force_updater.rb

```diff
@@ -42,154 +39,32 @@ def update_multiple_dependencies?
         end
 
         def force_update
-          in_a_temporary_bundler_context do
-            other_updates = []
-
-            begin
-              definition = build_definition(other_updates: other_updates)
-              definition.resolve_remotely!
-              specs = definition.resolve
-              dependencies_from([dependency] + other_updates, specs)
-            rescue ::Bundler::VersionConflict => e
-              raise unless update_multiple_dependencies?
-
-              # TODO: Not sure this won't unlock way too many things...
-              new_dependencies_to_unlock =
-                new_dependencies_to_unlock_from(
-                  error: e,
-                  already_unlocked: other_updates
-                )
-
-              raise if new_dependencies_to_unlock.none?
-
-              other_updates += new_dependencies_to_unlock
-              retry
-            end
-          end
-        rescue SharedHelpers::ChildProcessFailed => e
-          raise_unresolvable_error(e)
-        end
-
-        #########################
-        # Bundler context setup #
-        #########################
-
-        def in_a_temporary_bundler_context
           base_directory = dependency_files.first.directory
           SharedHelpers.in_a_temporary_repo_directory(base_directory,
```

Comment:
This can probably be switched over to use `in_a_native_bundler_context` as it is equivolent of lines 42-46 with unified bundler error handling, retrying etc.